### DISCUSSION TITLE
反作弊种子破解

### DISCUSSION THREAD

**Discussion by StaryyTraject** [community/community#79122] [Discussions]
### Select Topic Area

Question

### Body

我服务器需要一个反种子破解的插件

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @StaryyTraject ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
